package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Functionality commonly needed by Rollup plugins",
   "version": "1.5.2",
   "main": "dist/pluginutils.cjs.js",
-  "jsnext:main": "dist/pluginutils.es6.js",
+  "jsnext:main": "dist/pluginutils.es2015.js",
```

Comment:
Just es. Same what in `dist`. Better use `targets` option in config like here https://github.com/TrySound/rollup-plugin-eslint/blob/master/rollup.config.js
And duplicate `jsnext:main` in `module` field like here https://github.com/TrySound/rollup-plugin-eslint/blob/master/package.json#L5-L7